src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/StartupCode/StartupCodeHelpers.cs

```diff
@@ -18,6 +20,7 @@ internal static class StartupCodeHelpers
         internal static void Initialize()
         {
             InitializeStringTable();
+            RunEagerClassConstructors();
             RuntimeImports.RhEnableShutdownFinalization(0xffffffffu);
```

Comment:
We got rid of the shutdown finalization for .NET Core (https://github.com/dotnet/corefx/issues/5205 has the long story), so it would be more appropriate to delete the `RhEnableShutdownFinalization` here and keep the CORERT ifdef in Environment.cs as a reminder to clean it up for UWP as well.

The shutdown finalization as implemented in Redhawk runtime does not really work because of it does not suspend foreground threads like in full .NET. We never noticed because of UWP apps do not ever shutdown, and small unit tests do not highlight the problems.